holoviews/__init__.py

```diff
@@ -114,12 +113,7 @@
     HoloviewsUserWarning,
 )
 
-# Suppress warnings generated by NumPy in matplotlib
-# Expected to be fixed in next matplotlib release
```

Comment:
I don't know what the next matplotlib version was, but this was added nine years ago...